cache/caches.go

```diff
@@ -1,7 +1,7 @@
 package cache
 
 // CacheList is a list of caches for the dp-frontend-search-controller
-type CacheList struct {
+type List struct {
```

Comment:
Any particular reason why this has been changed to `List`? Isn't CacheList more explicit about what it is?
Also, update needed for the comment to the new name change